singularity/renewssl.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Getting environment variables from .env
+source ./loadenv.sh
+# Check if required variables are defined
+source ./varcheck.sh
+
+containerDataDir=$state
+if [ ! -d $containerDataDir ]; then
+    echo "$containerDataDir does not exist"
+    exit 1
+fi
+if [ ! -z ${DPDASH_SERVICE_HOST} ]
+then 
+    serviceHost=$DPDASH_SERVICE_HOST
+else
+    serviceHost=`hostname -f | xargs`
+fi
+cd ${containerDataDir}/ssl/ca
+
+echo 'Revoking existing certs'
+dirpath=${containerDataDir}/ssl/ca/certs
```

Comment:
For the record, I am putting the directory hierarchy here so `cd`s are better understood
```
ssl/
├── ca
│   ├── cacert.pem
│   ├── certs
│   │   ├── 01.pem
│   │   └── 02.pem
│   └── private
│       └── cakey.pem
├── client
│   ├── cert.pem
│   ├── key.pem
│   └── req.pem
├── mongo_client.pem
├── mongo_server.pem
└── server
    ├── cert.pem
    ├── key.pem
    └── req.pem
```

Reply:
Sure. BTW,  all lines below 27 are identical to here:

https://github.com/PREDICT-DPACC/dpdash/blob/585cb7cba46c2b3cd72432975340d8133dde47d6/singularity/init.sh#L51-L79